packages/asset-swapper/src/utils/market_operation_utils/index.ts

```diff
@@ -573,7 +572,10 @@ export class MarketOperationUtils {
         }
 
         // Generate a fallback path if required
-        await this._addOptionalFallbackAsync(side, inputAmount, optimalPath, dexQuotes, fills, opts, penaltyOpts);
+        // TODO(kimpers): Will experiment with disabling this and see how it affects revert rate
+        // to avoid yet another router roundtrip
+        // TODO: clean this up if we don't need it
+        // await this._addOptionalFallbackAsync(side, inputAmount, optimalPath, dexQuotes, fills, opts, penaltyOpts);
```

Comment:
We're gonna try skipping fallback orders for RFQT, as it seems like it would only marginally increase the revert rate, but decrease tx sticker shock for users, and remove one extra routing call for routes that include Native